modules/mapping-utils/src/createConnectionTypeDefs.js

```diff
@@ -4,8 +4,6 @@ export default ({ type, fields = '' }) => `
   type ${type.name} {
     mapping: JSON
 
-    extended(fields: [String]): JSON
```

Comment:
Is this on purpose?

Reply:
This is part of the normalization commit